versioning/BRM/extentmap.cpp

```diff
@@ -2406,20 +2363,20 @@ int ExtentMap::lookupLocalStartLbid(int      OID,
     grabEMEntryTable(READ);
     grabEMIndex(READ);
 
-    DBRootVec dbRootVec(std::move(getAllDbRoots()));
+    DBRootVec dbRootVec(getAllDbRoots());
```

Comment:
Nice one. This std::move prevents copy elision according with clangd suggestion.